src/components/newsFeed/news.js

```diff
@@ -11,18 +13,21 @@ class News extends React.Component {
     };
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  formatDate(t, timestamp) {
+    const hours = Math.floor(Math.abs(new Date() - new Date(timestamp)) / 36e5);
+    if (hours <= 1) return t('just now');
+    if (hours > 1 && hours <= 24) return t('{{hours}}h ago', { hours });
+    if (hours > 24 && hours <= 48) return t('Yesterday');
```

Comment:
"Yesterday" Is missing from `i18n/locales/de/common.json`

![Screenshot 2019-09-06 at 10 22 30](https://user-images.githubusercontent.com/1254342/64412846-98dd0380-d090-11e9-8e15-b8cbfba473cf.png)